core/test-api/collection/src/main/java/org/opennms/core/collection/test/JUnitCollectorExecutionListener.java

```diff
@@ -154,7 +154,16 @@ public void afterTestMethod(TestContext testContext) throws Exception {
             }
         }
 
-        FileUtils.deleteDirectory(m_snmpRrdDirectory);
+        try {
+            FileUtils.deleteDirectory(m_snmpRrdDirectory);
+        } catch (Exception deletionException) {
```

Comment:
Can we be more specific in the exception we catch here? Would catching an IOException have the same affect?

Reply:
Yes, IOException is sufficient here.
I had missed the line below which requires RuntimeException.
Fixed.